service/provider/src/main/java/gov/va/vro/service/provider/MasOrderExamProcessor.java

```diff
@@ -42,7 +49,15 @@ public void process(Exchange exchange) {
           MAS_CONDITION_CODES.getOrDefault(claimPayload.getDiagnosticCode(), "NA"));
       masOrderExamRequest.setConditions(List.of(masOrderExamConditions));
       var response = masApiService.orderExam(masOrderExamRequest);
-      log.info("Order Exam Response :  " + response);
+      log.info("Order Exam Response :  " + response + " saving as ORDER_SUBMITTED");
+      ExamOrder examOrder =
+          ExamOrder.builder()
+              .collectionId(Integer.toString(claimPayload.getCollectionId()))
+              .idType(MasAutomatedClaimPayload.CLAIM_V2_ID_TYPE)
+              .status(SUBMITTED_STATUS)
+              .examOrderDateTime(OffsetDateTime.now())
```

Comment:
If I understand correctly, the `examOrderDateTime` should be populated with the datetime from MAS when it successfully orders the exam. This place in the code represents when VRO requests the exam to be ordered.
```suggestion
              .examOrderDateTime(OffsetDateTime.now())
```